affiliates_and_sponsors.md

```diff
@@ -29,12 +18,30 @@ title: affiliates and sponsors
 </p>
 
 <h2>Additional Sponsors</h2>
+
 <p>
-<span class="founding"><img class="sponsor" src="/img/sponsors/vmware.png" /></span>
+<img class="sponsor" src="/img/sponsors/ethereum.png" />
 </p>
 
+
 </div>
 
+
+# Founding Sponsors
+<h4> Platinum </h4> 
+<dl>
+  <dd>Amazon </dd>
+</dl>
+<h4> Silver </h4>
+<dl>
+<dd> Adacore, Certora </dd>
+</dl>
+<h4> Additional </h4>
+<dl>
+<dd> Vmware </dd>
+</dl>
+
```

Comment:
Let's just have a bulleted list without the categories (i.e., Platinum, Silver, etc.)